packages/admin/cms-admin/src/dam/FileForm/CropSettingsFields.tsx

```diff
@@ -9,6 +9,15 @@ import { FormattedMessage } from "react-intl";
 import { ChooseFocalPoint } from "../../common/image/ChooseFocalPoint";
 import { EditImageFormValues } from "./EditFile";
 
+const CroppingInfoBox = styled(Box)`
+    margin-top: ${({ theme }) => theme.spacing(2)};
+    padding-left: ${({ theme }) => theme.spacing(2)};
+
+    & > p:first-of-type {
```

Comment:
I'd rather use `gutterBottom` of the Typography component for this

Reply:
I used `paragraph` instead. It also adds a bit of margin but not as much as `gutterBottom`